### ISSUE TITLE
ReferenceError: site_id_email_uniqueCompoundUniqueInput is not defined

### ISSUE THREAD

**Issue by s-kris** [MichalLytek/typegraphql-prisma#109]
**Describe the Bug**
The graphql server failes to run with the error: `ReferenceError: site_id_email_uniqueCompoundUniqueInput is not defined`

**To Reproduce**
Create a Prisma model with the compound key of site_id + email 

**Expected Behavior**
Graphql API should run normally 

**Logs**
```
[nodemon] starting `ts-node --project tsconfig.server.json server/index.ts`

/app_path/node_modules/@generated/type-graphql/resolvers/inputs/site_member_invitationsSite_member_invitations.site_id_email_uniqueCompoundUniqueInput.js:38
site_id_email_uniqueCompoundUniqueInput;
^
ReferenceError: site_id_email_uniqueCompoundUniqueInput is not defined
    at Object.<anonymous> (/app_path/node_modules/@generated/type-graphql/resolvers/inputs/site_member_invitationsSite_member_invitations.site_id_email_uniqueCompoundUniqueInput.js:38:1)
    at Module._compile (node:internal/modules/cjs/loader:1092:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1121:10)
    at Object.require.extensions.<computed> [as .js] (/usr/local/lib/node_modules/ts-node/src/index.ts:1032:44)
    at Module.load (node:internal/modules/cjs/loader:972:32)
    at Function.Module._load (node:internal/modules/cjs/loader:813:14)
    at Module.require (node:internal/modules/cjs/loader:996:19)
    at require (node:internal/modules/cjs/helpers:92:18)
    at Object.<anonymous> (/app_path/node_modules/@generated/type-graphql/resolvers/inputs/Site_member_invitationsWhereUniqueInput.js:33:98)
    at Module._compile (node:internal/modules/cjs/loader:1092:14)
[nodemon] app crashed - waiting for file changes before starting...
```

**Environment (please complete the following information):**

- OS: MacOS 10.15.7
- Node: 15.14.0
- `typegraphql-prisma` version: 0.14.0
- Prisma 2 version: 2.21.2
- TypeScript version: 4.1.3

**Additional Context**
I'm using Prisma with Next.js (custom server mode) with API routes already. 
I just wanted to try typegraphql as I'm planning to move to graphql. 
nextjs: 10.1.3

**Comment by MichalLytek** [comment]
> Create a Prisma model with the compound key of site_id + email

Please paste a Prisma schema to reproduce reported issue

**Comment by s-kris** [comment]
Sure. Here is it:

```
generator client {
  provider = "prisma-client-js"
}

generator typegraphql {
  provider = "typegraphql-prisma"
}

datasource db {
  provider = "postgresql"
  url      = env("DATABASE_URL")
}

model site_member_invitations {
  id         Int      @id @default(autoincrement())
  site_id    Int
  email      String
  created_at DateTime @default(now())
  updated_at DateTime @default(now())
  invited_by Int

  @@unique([site_id, email], name: "site_member_invitations.site_id_email_unique")
}
```

**Comment by MichalLytek** [comment]
Duplicate of #77 🔒
